web/static/js/consent.js

```diff
@@ -0,0 +1,48 @@
+module.exports = (function() {
+  var noConsent = document.querySelectorAll(".no-consent");
+  var yesConsent = document.querySelectorAll(".yes-consent");
+
+  noConsent.forEach(function(el){
+    el.addEventListener("click", function (e) {
+      var index = e.target.id[e.target.id.length - 1];
+      document.getElementById("error-consent-" + index).classList.remove("dn");
```

Comment:
https://developer.mozilla.org/en/docs/Web/API/Element/classList. `classList.remove` doesn't have IE support.

Maybe `...className = ...className.replace(' dn', '')` or something like that